queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon%';
+select name from animals where date_of_birth between (to_date('01/01/2016','mm/dd/yyyy')) and (to_date('12/31/2019','mm/dd/yyyy'));
+select name from animals where neutered = true and escape_attempts < 3;
+select date_of_birth from animals where name = 'Agumon' or name= 'Pikachu';
+select name,escape_attempts from animals where weight_kg > 10.5;
+select * from animals where neutered = true;
+select * from animals where name != 'Gabumon';
+select * from animals where weight_kg >= 10.4 and weight_kg <= 17.3;
```

Comment:
- [ ] Let us use plural form while we are answering for instance `SELECT`, `FROM` and `WHERE` to be in plural 
- [ ] Again your queries should throw an error because you are selecting the column names that do not exist, check the table you created with the column names, you have created columns using the First capital letter.  `Escape_attempts` ` Neutered`

Reply:
Thank you for the remarks. Done

Reply:
Well done